ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Mempool.hs

```diff
@@ -283,22 +284,22 @@ theLedgerLens f x =
   Tx Limits
 -------------------------------------------------------------------------------}
 
-instance (SL.PraosCrypto c) => TxLimits (ShelleyBlock (ShelleyEra c)) where
+instance (ShelleyBasedEra (MaryEra c)) => TxLimits (ShelleyBlock (ShelleyEra c)) where
   type TxMeasure (ShelleyBlock (ShelleyEra c)) = ByteSize
   txMeasure        = ByteSize . txInBlockSize . txForgetValidated
   txsBlockCapacity = ByteSize . txsMaxBytes
 
-instance (SL.PraosCrypto c) => TxLimits (ShelleyBlock (AllegraEra c)) where
+instance (ShelleyBasedEra (MaryEra c)) => TxLimits (ShelleyBlock (AllegraEra c)) where
```

Comment:
Copy-and paste error, `MaryEra`?

Reply:
Whoops, not sure how I managed that! Good catch